LibreOfficeWriter_Char.au3

```diff
@@ -841,19 +740,15 @@ EndFunc   ;==>_LOWriter_CharStyleGetObj
 ;				   @Error 1 @Extended 7 Return 0 = $sParentStyle Doesn't exist in this Document.
 ;				   @Error 1 @Extended 8 Return 0 = $bHidden not a Boolean.
 ;				   --Property Setting Errors--
-;				   @Error 4 @Extended ? Return 0 = Some settings were not successfully set. Use BitAND to test @Extended for
-;				   +								the following values: 1, 2, 4
+;				   @Error 4 @Extended ? Return 0 = Some settings were not successfully set. Use BitAND to test @Extended for the following values: 1, 2, 4
```

Comment:
Please use
````
the following values:
```
instead:
```
the following values: 1, 2, 4
```

there is no need to enumarate values when you have them listed below.